.mergify.yml

```diff
@@ -1,46 +1,34 @@
 pull_request_rules:
-  - name: 'rebase unreviewed non-release PRs'
+  - name: "rebase unreviewed non-release PRs"
     conditions:
       - head~=^(?!(release|hotfix)).*$
```

Comment:
As discussed, only try to convert this to a (double-quoted) string to. Then all conditions are specified in the same manner.